test/unit/transforms/code-scanning-alert.test.ts

```diff
@@ -50,14 +50,22 @@ describe("code_scanning_alert transform", () => {
 			...codeScanningPayload,
 			alert: {...codeScanningPayload.alert, rule: {...codeScanningPayload.alert.rule, description: "A".repeat(300)}}
 		};
-		const remoteLinks = transformCodeScanningAlert(buildContext(payload));
+		const remoteLinks = await transformCodeScanningAlert(buildContext(payload));
 		expect(remoteLinks.remoteLinks[0].description).toHaveLength(255);
 	})
 
-	// TODO ARC-618
 	it("code_scanning_alert with pr reference queries Pull Request title", async () => {
 		const payload = {...codeScanningPayload, ref: "refs/pull/8/merge"};
-		const remoteLinks = transformCodeScanningAlert(buildContext(payload));
-		expect(remoteLinks.remoteLinks[0].associations[0].values[0]).toEqual("GH-8");
+		const context = buildContext(payload);
+		const mySpy = jest.spyOn(context.github.pulls, "get");
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		(mySpy as jest.MockInstance<any, any>).mockResolvedValue({
```

Comment:
I need this `(mySpy as jest.MockInstance<any, any>)` so that I don't have to mock a complete `PullRequestResponse` that `context.github.pulls.get` returns. (notice I only return an object `{title: x}` instead of needing every single attribute of a pullrequest)